lumigator/python/mzai/backend/backend/tests/conftest.py

```diff
@@ -47,6 +57,48 @@ def format_dataset(data: list[list[str]]) -> str:
     return buffer.read()
 
 
```

Comment:
let's make this a fixture so we can load these into other modules correctly and share across files